jenkins-apb/README.md

```diff
@@ -0,0 +1,28 @@
+jenkins-apb
+======================
+
+## What it does
+This APB deploys a Jenkins server capable of managing OpenShift Pipeline builds and supporting OpenShift-based
+oauth login.
+
+## Requirements
+
+## Parameters
+* persistent: Optional, if true use a persistent volume for Jenkins data.
```

Comment:
@codificat I was thinking in a later PR to use plans.  Unless the team thinks otherwise and I can always modify it.